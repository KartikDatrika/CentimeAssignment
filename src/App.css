.App {
  height:100%;
}
body {
  font-family: 'KoHo', sans-serif;
  background-color: #282c34;
}
.columnName {
    color: white;
    font-weight: 600;
    font-size: 30px;
}

.table-container {
    max-height: 500px;
    overflow-y: scroll;
}

