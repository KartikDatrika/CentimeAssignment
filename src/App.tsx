import { Provider } from 'react-redux';
import './App.css';
import ExpenditureVisualization from './Components/ExpenditureVisualization/ExpenditureVisualization';
import configureStore from './State-Management/configureStore';

function App() {
  const store = configureStore();
  return (
    <Provider store={store}>
    <div className="App">
      <header className="text-center mb-5">
       <img  src={"https://centime.com/images/logo.png"} alt="logo" />
      </header>
        <ExpenditureVisualization/>
    </div>
    </Provider>
  );
}

export default App;
