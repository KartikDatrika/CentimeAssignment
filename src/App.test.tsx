import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import './Localization/i18n';

describe('App Test Suite', () => {
  it('App is rendering without any error !', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });

  it('App is rendering with Centime logo !', () => {
    const { getAllByAltText } = render(<App />);
    expect(getAllByAltText('logo')).toBeDefined();
  });
})
