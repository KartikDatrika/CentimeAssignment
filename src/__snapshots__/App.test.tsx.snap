// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test Suite App is rendering without any error ! 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="text-center mb-5"
    >
      <img
        alt="logo"
        src="https://centime.com/images/logo.png"
      />
    </header>
    <div
      class="container"
    >
      <div
        class="mb-5"
      >
        <label
          class="columnName"
        >
          Translate to
        </label>
        <select
          class="form-control"
          data-testid="select"
        >
          <option
            data-testid="select-option"
            value="en"
          >
            English
          </option>
          <option
            data-testid="select-option"
            value="af"
          >
            Afrikaans
          </option>
          <option
            data-testid="select-option"
            value="bg"
          >
            Bulgarian
          </option>
          <option
            data-testid="select-option"
            value="hr"
          >
            Croatian
          </option>
          <option
            data-testid="select-option"
            value="cs"
          >
            Czech
          </option>
          <option
            data-testid="select-option"
            value="da"
          >
            Danish
          </option>
          <option
            data-testid="select-option"
            value="nl"
          >
            Dutch
          </option>
          <option
            data-testid="select-option"
            value="fr"
          >
            French
          </option>
          <option
            data-testid="select-option"
            value="de"
          >
            German
          </option>
          <option
            data-testid="select-option"
            value="el"
          >
            Greek
          </option>
          <option
            data-testid="select-option"
            value="hi"
          >
            Hindi
          </option>
          <option
            data-testid="select-option"
            value="ja"
          >
            Japanese
          </option>
          <option
            data-testid="select-option"
            value="ko"
          >
            Korean
          </option>
          <option
            data-testid="select-option"
            value="ru"
          >
            Russian
          </option>
          <option
            data-testid="select-option"
            value="sr"
          >
            Serbian
          </option>
          <option
            data-testid="select-option"
            value="sk"
          >
            Slovak
          </option>
          <option
            data-testid="select-option"
            value="es"
          >
            Spanish
          </option>
          <option
            data-testid="select-option"
            value="sv"
          >
            Swedish
          </option>
          <option
            data-testid="select-option"
            value="th"
          >
            Thai
          </option>
          <option
            data-testid="select-option"
            value="tr"
          >
            Turkish
          </option>
          <option
            data-testid="select-option"
            value="vi"
          >
            Vietnamese
          </option>
        </select>
      </div>
    </div>
  </div>
</div>
`;
